CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [next]
 
+- fix(Text): `_getFontDeclaration` [#9082](https://github.com/fabricjs/fabric.js/pull/9082)
```

Comment:
This changelog doesn't tell much about the change. It should be more `_getFontDeclaration: passed style should take precedence` as the test case.

Reply:
point noted

Reply:
Yeah i also changed the pr description and the merge commit, but forgot to update the changelog.